Add note about setting primary sw type (#488)
memfault/memfault-docs: docs/platform/software-version-hardware-version.mdx

the individual Device page, as well as when filtering for the device, and how it
appears in various fleet-level charts.

> To have your software version appear in the Devices list (and in the detailed
> Device view) it needs to be added to the Primary Software Version under Settings
> –> Hardware –> Create Hardware Version.
> 

Reviewer: It sounds to me like a repeat of the previous paragraph with the location of the configuration option spelled out.

How about making this simpler?

```suggestion
You can configure the **Primary Software Type** in Settings
–> Hardware –> Create Hardware Version.

```

Author: Beautiful, thank you!
